ScanbotBarcodeSDKExample/BarcodeSDK.MAUI.Example/Pages/BarcodeClassicComponentPage.xaml.cs

```diff
@@ -40,13 +44,22 @@ protected override void OnDisappearing()
 
             // Stop barcode detection manually
             cameraView.StopDetection();
-            
+
             cameraView.Handler?.DisconnectHandler();
         }
 
         private void CameraView_OnOnBarcodeScanResult(RTU.v1.BarcodeResultBundle result)
         {
             HandleScannerResults(result);
         }
+
+        void ToggleCameraPreview_Clicked(System.Object sender, System.EventArgs e)
+        {
+            MainThread.InvokeOnMainThreadAsync(() =>
```

Comment:
@scan-mayank, is this needed? Isn't the code already running on the UI thread?

Reply:
Removed and tested